unstructured/partition/html.py

```diff
@@ -4,6 +4,7 @@
 
 from unstructured.documents.elements import Element
 from unstructured.documents.html import HTMLDocument
+from unstructured.partition.xml import VALID_PARSERS
```

Comment:
I think this may need to be `from unstructured.documents.xml import VALID_PARSERS`

Reply:
Done, thank you!
I also noticed that I forgot to add the description in the docstring, fixed that too.